signinapp/model.py

```diff
@@ -228,16 +245,16 @@ def total_funds(self) -> str:
 
     @staticmethod
     def get_visible_users() -> list[User]:
-        return db.session.scalars(select(User).where(User.role.has(visible=True)))
+        return db.session.scalars(select(User).where(User.role_names == "visible"))
```

Comment:
Does this work when the user has multiple roles?

Reply:
Surprisingly yes - I was confused by it too, but this is how it was shown in the sqlalchemy docs